modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js

```diff
@@ -175,6 +175,7 @@ function FieldBase({
 	onClick,
 	overMaximumRepetitionsLimit = false,
 	readOnly,
+	renderInputEditedFlag = true,
```

Comment:
I think if you flip the  logic you would get a better design, once its as [boolean prop](https://dzone.com/articles/react-guide-to-props-part-i) and the default behavior is to show the input:
```suggestion
	hideEditedFlag,
```
So that you can set attribute on omitting its value:
```javascript
// Displays edited input
<FieldBase>...</FieldBase>

// Hides edited input
<FieldBase hideEditedFlag>...</FieldBase>

```

Reply:
Oh. Looks good. Thank you for the tip.